tests/acceptance/features/webUIPasswordChangeUsersPage/passwordChangeLowercaseLetters.feature

```diff
@@ -0,0 +1,35 @@
+@webUI @insulated @disablePreviews @skipOnFIREFOX
+Feature: enforce the required number of lowercase letters in a password on the password change from users page
```

Comment:
`enforce the required number of lowercase letters for the user's password in the users page`